packages/@webex/plugin-meetings/src/roap/index.ts

```diff
@@ -191,64 +191,62 @@ export default class Roap extends StatelessWebexPlugin {
       headers: ['includeAnswerInHttpResponse', 'noOkInTransaction'],
     };
 
-    // When reconnecting, it's important that the first roap message being sent out has empty media id.
-    // Normally this is the roap offer, but when TURN discovery is enabled,
-    // then this is the TURN discovery request message
-    return this.turnDiscovery.isSkipped(meeting).then((isTurnDiscoverySkipped) => {
-      const sendEmptyMediaId = reconnect && isTurnDiscoverySkipped;
+    // The only time we want to send an empty media id is when we are reconnecting, because this way we tell Locus
+    // that it needs to create a new confluence, but when reconnecting we always send TURN_DISCOVERY_REQUEST first,
+    // so we don't need to ever send an empty media id here
+    const sendEmptyMediaId = false;
```

Comment:
this will always be false no? what's the point adding it here?

Reply:
as far as i can tell this method is the generic sendRoapMediaRequest regardless if it's a reconnection or connection

Reply:
yeah I could remove it, I left it so that it's kind of more explicit and easier to understand and also if potentially we change the logic again in the future about when to do TURN discovery when reconnecting then we might need to send empty media Id here